crypto/src/main/java/org/cryptimeleon/incentive/crypto/model/JoinFirstStepOutput.java

```diff
@@ -6,9 +6,15 @@
 import lombok.Value;
 import org.cryptimeleon.incentive.crypto.IssueJoinRandomness;
 
+/**
+ * Wrapper class for the output the user generates from the first step of the Join algorithm
+ * (which is part of the Issue-Join protocol).
+ * Consists of the randomness that the user derives from the PRF
```

Comment:
Does he derive the join randomness from the PRF? I don't think he should.

Reply:
Of course he does not, thanks for pointing out